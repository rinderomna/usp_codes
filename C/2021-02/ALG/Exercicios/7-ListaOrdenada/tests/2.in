REMOVE 3
INSERE 3 Tulio Carlos
REMOVE 3
INSERE 3 Marieta Antonia
INSERE 3 Jose Silva
REMOVE 2
IMPRIMIR

