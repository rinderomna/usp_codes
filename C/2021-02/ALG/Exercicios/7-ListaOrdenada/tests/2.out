INVALIDO
INVALIDO
INVALIDO
3, Marieta Antonia;
